supervisor/api/supervisor.py

```diff
@@ -115,6 +117,7 @@ async def info(self, request: web.Request) -> Dict[str, Any]:
             ATTR_DEBUG_BLOCK: self.sys_config.debug_block,
             ATTR_DIAGNOSTICS: self.sys_config.diagnostics,
             ATTR_CONTENT_TRUST: self.sys_config.content_trust,
+            ATTR_FORCE_SECURITY: self.sys_config.force_security,
```

Comment:
`force_security` can be lot of things, how about `mandatory_content_trust`?

Reply:
It's not all about that. We have other places which we need make decisions like pwnd flows. It's allowed to use insecure secrets? Ect. With enabling we handle security issues straight forward and that can have negative side effects if you are not aware

Reply:
I add a new commit to show the plan